corehq/ex-submodules/phonelog/utils.py

```diff
@@ -120,6 +120,7 @@ def _process_user_error_subreport(domain, xform):
             msg=error['msg'],
             session=error['session'],
             type=error['type'],
+            context_node=error.get('context_node', ''),
```

Comment:
Note that unlike the other fields, this doesn't fail hard if the field is missing.